diff --git a/app/backend/cloudflare-tunnel.js b/app/backend/cloudflare-tunnel.js
index 9426665..2a2c31f 100644
--- a/app/backend/cloudflare-tunnel.js
+++ b/app/backend/cloudflare-tunnel.js
@@ -73,7 +73,12 @@ class CloudflaredTunnel {
         ];
 
         if (imp_err_ids.some(subStr => msg.includes(subStr))) {
-            throw new Error(msg.split('ERR')[1].trim());
+            console.error('------------------------------------------------');
+            this.stop();
+            setTimeout(() => {
+                this.start();
+            }, 5000);
+/*             throw new Error(msg.split('ERR')[1].trim()); */
         }
 
     }
